common/envOverwrite/overwriter.go

```diff
@@ -1,71 +1,150 @@
 package envOverwrite
 
-import "strings"
+import (
+	"strings"
 
-// EnvValues is a map of environment variables odigos uses for various languages and goals.
+	"github.com/odigos-io/odigos/common"
+)
+
+type envValues struct {
+	delim string
+	values map[common.OtelSdk]string
+}
+// EnvValuesMap is a map of environment variables odigos uses for various languages and goals.
 // The key is the environment variable name and the value is the value to be set or appended
 // to the environment variable. We need to make sure that in case any of these environment
 // variables is already set, we append the value to it instead of overwriting it.
-var EnvValues = map[string]struct {
-	Value string
-	Delim string
-}{
+var EnvValuesMap = map[string]envValues{
```

Comment:
perhaps add a note that these values should be correlated to the ones in the dockerfile